[feature] manage owner, group, and permission on sasl2.db

trombik opened the pull request:
fixes #10

Commit:
merge issue_3 #8

Commit:
manage owner, group, and permission on sasl2.db

fixes #10

Commit:
Merge branch 'master' into issue_10

Commit:
fix broken merge

Commit:
remove duplicated variable, warned by ansible

mitsururike commented on tasks/main.yml:
```diff
@@ -37,6 +37,25 @@
+- name: Set cyrus_sasl_sasldb_file_real_path
+  set_fact:
+    # BSDs appends '.db' to the given file name, but others treat it as-is
+    cyrus_sasl_sasldb_file_real_path: "{% if ansible_os_family == 'FreeBSD' or ansible_os_family == 'OpenBSD' %}{{ cyrus_sasl_sasldb_file }}.db{% else %}{{ cyrus_sasl_sasldb_file }}{% endif %}"
```
need an explanation why adding .db to __cyrus_sasl_sasldb_file in vars/*BSD.yml does not work.
'cyrus_sasl_saslpassword_command' and '/usr/local/bin/sasl_check_pw' works without .db extension?

trombik replied:
I need the explanation, too, that who decide to implement the BSD behavior. as the comment says, "$command $path" means the actual file name is "$path.db" on BSD implementations, which has been known so for years. see [dbm(3)](https://www.freebsd.org/cgi/man.cgi?dbm(3)).

as for your question, yes, as long as the library is BSD-derived one.

mitsururike reviewed (commented):
* if you use 'sasl2.db' as a generic name of sasl db file, it is better s/sasl2.db/sasldb file/g.   'sasl2.db' looks like a real file name.
* s/sasldb_group_permission/sasldb_permission/g because they are permission, not group permission.

Commit:
s/cyrus_sasl_sasldb_group_permission/cyrus_sasl_sasldb_file_permission/g

as suggested in
https://github.com/reallyenglish/ansible-role-cyrus-sasl/pull/12#pullrequestreview-35030935

Commit:
add a reference
